### DISCUSSION TITLE
Open files as other extensions? Open .twig files as .html

### DISCUSSION THREAD

**Discussion by estebancastro** [zed-industries/zed#9679] [Feature Requests]
There's no support yet for .twig files? so I need open all .twig files as .html to get at least highlight on the html and not see everything plain like this:

<img width="718" alt="Screenshot 2024-03-21 at 10 35 29 PM" src="https://github.com/zed-industries/zed/assets/2717274/c9ad6a09-07d6-4349-9cfb-26dc4a33519d">

There's a way on Sublime with "Open with the current extension as..."

<img width="440" alt="Screenshot 2024-03-21 at 10 34 57 PM" src="https://github.com/zed-industries/zed/assets/2717274/ddfbdc82-316d-4431-ba4c-cb5f85ce9677">

**Comment by 0x2CA** [comment]
![image](https://github.com/zed-industries/zed/assets/19868292/539a1478-445a-4a74-9bc2-c075795dbcfe)
#9290

**Reply by estebancastro** [reply]
thanks, but it doesn't make any difference, still "unknow".

<img width="640" alt="Screenshot 2024-03-22 at 11 59 06 AM" src="https://github.com/zed-industries/zed/assets/2717274/3b232b35-4299-4326-b355-fc41aac16497">

I can't find that in the docs by the way.

my config:

```
{
    "project_panel": {
        "dock": "left"
    },
    "collaboration_panel": {
        "dock": "right"
    },
    "theme": "One Dark",
    "telemetry": {
        "diagnostics": false,
        "metrics": false
    },
    "ui_font_size": 15,
    "buffer_font_size": 15,
    "tab_size": 4,
    "formatter": "language_server",
    "file_types": {
        "HTML": [
            "twig"
        ]
    }
}
```

**Reply by baldwindavid** [reply]
Are you using a preview or stable release? This was just released in Preview (https://github.com/zed-industries/zed/releases/tag/v0.128.0-pre) and won't be in Stable until next Wednesday.

**Comment by estebancastro** [comment]
> Are you using a preview or stable release? This was just released in Preview (https://github.com/zed-industries/zed/releases/tag/v0.128.0-pre) and won't be in Stable until next Wednesday.

oh! I see, that is, I'm using the stable (0.127.4), I can wait to the next wednesday, thanks.
